pkgs/applications/graphics/drawing/default.nix

```
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, file
+, gobject-introspection
+, pkgconfig
+, appstream-glib
+, meson
+, ninja
+, python3
+, gtk3
+, wrapGAppsHook
+, gettext
+, glib
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drawing";
+  version = "0.4.10";
+
+  src = fetchFromGitHub {
+    owner = "maoschanz";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1i8b7hkv9484c0pr2lag6xkf4mq5b2gwcawppgfkf1l8971zxqik";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+    pkgconfig
+    appstream-glib
+    meson
+    ninja
+    python3
+    gtk3
+    gettext
+    glib
+    python3Packages.wrapPython
+  ];
+
+  buildInputs = [
+    file
+    desktop-file-utils
+    gtk3
+    python3
+    glib
+    gobject-introspection
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pygobject3
+  ];
+
+  postPatch = ''
+    chmod +x build-aux/meson/postinstall.py # patchShebangs requires executable file
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  postFixup = ''
+    wrapPythonPrograms
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A free basic image editor, similar to Microsoft Paint, but aiming at the GNOME desktop.";
+    homepage = "https://maoschanz.github.io/drawing/";
+    maintainers = with maintainers; [ mothsart ];
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```

see https://github.com/maoschanz/drawing/blob/eb2397bccac38c8d671d33a66a8bcc8de5352927/src/message_dialog.py#L8

---

not pay attention to the nuance : good review !